common/config/babel/.babelrc.js

```diff
@@ -23,6 +23,8 @@ process.env['COMMUNICATION_REACT_FLAVOR'] === 'stable' &&
         'file-sharing',
         // Adhoc calls to a Teams user.
         'teams-adhoc-call',
+        // props to allow Contoso to overwrite timestamp format for chat messages, one in locale and one in message thread component
+        'dateTimeCustomization',
```

Comment:
Can we rename this to match the pattern we've used in other features: `date-time-customization`.